docs/feature_mouse_keys.md

```diff
@@ -47,12 +47,15 @@ You can see an example in the `_ML` here: https://github.com/qmk/qmk_firmware/bl
 The default speed for controlling the mouse with the keyboard is intentionally slow. You can adjust these parameters by adding these settings to your keymap's `config.h` file. All times are specified in milliseconds (ms).
 
 ```
-#define MOUSEKEY_DELAY             300
-#define MOUSEKEY_INTERVAL          50
-#define MOUSEKEY_MAX_SPEED         10
-#define MOUSEKEY_TIME_TO_MAX       20
-#define MOUSEKEY_WHEEL_MAX_SPEED   8
-#define MOUSEKEY_WHEEL_TIME_TO_MAX 40
+#define MOUSEKEY_DELAY                   300
+#define MOUSEKEY_INTERVAL                50
+#define MOUSEKEY_MAX_SPEED               10
+#define MOUSEKEY_TIME_TO_MAX             20
+#define MOUSEKEY_WHEEL_MAX_SPEED         8
+#define MOUSEKEY_WHEEL_TIME_TO_MAX       40
+#define MOUSEKEY_ACCEL0_SPEED(max_speed) max_speed / 4
```

Comment:
Using `max_speed` may not be the best option here.  

It may be best to use just the number here.